Mathlib/RingTheory/TensorProduct.lean

```diff
@@ -437,17 +437,18 @@ instance : AddMonoidWithOne (A ⊗[R] B) :=
 instance : AddCommMonoid (A ⊗[R] B) := by infer_instance
 
 -- providing this instance separately makes some downstream code substantially faster
-instance instMul : Mul (A ⊗[R] B) where
-  mul a b := mul a b
+instance instMul : Mul (A ⊗[R] B) := ⟨fun a b ↦ mul a b⟩
```

Comment:
Presumably this change is not needed?

Reply:
No. I am blunt instrument. I'll restore it.

Reply:
Done.